modules/arm_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2018 Intel Corporation
```

Comment:
Let's update years in all files including source code. I propose `2020-2021`

Reply:
fixed